base/util.jl

```diff
@@ -220,6 +220,45 @@ macro allocated(ex)
     end
 end
 
+function totalsizeof(x, ptr_cache = Set())
+    if in(pointer_from_objref(x), ptr_cache)
+        return 0
+    end
+    push!(ptr_cache, pointer_from_objref(x))
+
+    sz = 0
+    try
+        sz = sizeof(x)
```

Comment:
It fails on the Symbol type. An Error is raised that says "value does not have a canonical binary representation."